### PR TITLE
add more number of requested memzone segments to set memzone_max manu…

### PR DESCRIPTION
**Pull request by jianbingzhu**
add more number of requested memzone segments to set memzone_max manually for 1K IP camera sessions

### CODE REVIEW

File: media-proxy/src/proxy_context.cc [hunk 1]
```diff
@@ -200,6 +201,7 @@ void ProxyContext::ParseStInitParam(const mcm_conn_param* request, struct mtl_in
     st_param->rx_queues_cnt[MTL_PORT_P] = 32;
     st_param->tx_queues_cnt[MTL_PORT_P] = 32;
     st_param->lcores = NULL;
+    st_param->memzone_max = 3000;
```
Comment (qiang-han) [inline media-proxy/src/proxy_context.cc:204]: miss the declaration of "memzone_max".

**Comment by qiang-han** [comment]
need to rebase to the latest source code.

File: media-proxy/src/proxy_context.cc (hunk 1 above)
Comment (jianbingzhu) [reply media-proxy/src/proxy_context.cc:204]: The memzone_max has been declared in mtl_api.h and only need to update iMTL, which has been in https://github.com/OpenVisualCloud/Media-Transport-Library/pull/671.

**Review by qiang-han — APPROVED** [review]
LGTM

**Pull request merged**